doc/10-icinga-template-library.md

```diff
@@ -922,6 +922,7 @@ load_cload1     | **Optional.** The 1-minute critical threshold. Defaults to 10.
 load_cload5     | **Optional.** The 5-minute critical threshold. Defaults to 6.
 load_cload15    | **Optional.** The 15-minute critical threshold. Defaults to 4.
 load_percpu     | **Optional.** Divide the load averages by the number of CPUs (when possible). Defaults to false.
+load_procs_to_show | **Optional** Number of processes to show when printing the top consuming processes. (Default value is 0)
```

Comment:
Please reformat the Markdown table to have the separating line at the same position, adding some spaces (or dashes for the line) to the other entries.